### DISCUSSION TITLE
Problem with Symmetrization of tensors

### DISCUSSION THREAD

**Discussion by SamapanB** [FeynCalc/feyncalc#142] [Q&A] [answered]
Hi, I am using Mathematica 12.3 and the FeynCalc version is 9.3.1.

I have a tensor, using which I want to write a symmetrized version of it. e.g. - if I have, f(x,y), upon symmetrization, I would like to have, f(x,y) + f(y,x). I found the function 'Symmetrize' for this purpose as described in https://www.feyncalc.org/FeynCalcBook/Symmetrize/index.html

But when using it on my computer, it doesn't work. I get the following:

In[1] Symmetrize[f[a, b], {a, b}]
Out[1] Symmetrize[f(a,b),{a,b}]

whereas I expected :
(1/2)(f(a,b)+f(b,a))

I don't understand this function 'Symmetrize' works. The Mathematica database does not mention anything like the above website. Has the functionality of 'Symmetrize' as mentioned in the above website changed? Is there a way out to achieve what I am looking for?

**Comment by vsht** [comment, marked as answer]
Hi, 

that documentation page is completely outdated. You should use

https://feyncalc.github.io/reference for the stable version

and

https://feyncalc.github.io/referenceDev for the dev version

The function has been renamed to `FCSymmetrize` since ages 

https://feyncalc.github.io/FeynCalcBook/ref/FCSymmetrize.html

**Reply by SamapanB** [reply]
Thanks
